fendermint/app/src/cmd/key.rs

```diff
@@ -66,6 +70,27 @@ cmd! {
   }
 }
 
+cmd! {
+  AddPeer(self) {
+    let node_key = NodeKey::load_json_file(&self.node_key_file).unwrap();
+    let peer_id : String = node_key.node_id().to_owned().to_string()+"@"+self.network_addr.as_str();
+    let peers = std::fs::read_to_string(&self.local_peers_file).context("failed to read peer file");
+    match peers {
+        Ok(mut v) => {
+            if v.is_empty()  {
+                std::fs::write(&self.local_peers_file, peer_id)?;
+            } else {
+                v = v.to_owned().to_string()+ "," + peer_id.as_str();
+                std::fs::write(&self.local_peers_file, v)?;
+            }
+        }
+        Err(_) => {
+                std::fs::write(&self.local_peers_file, peer_id)?;}
```

Comment:
What's that `}` at the end? 

It looks like you haven't configured your editor to use `rustfmt`. CI will reject unformatted code.

Reply:
Ah, I remember, because we are inside `cmd!` macro it doesn't do formatting :bulb: 
That's why I started delegating out, to make it uniform. 

Thanks for fixing.